### PR TITLE
Update Error handling

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/media-browser/hui-dialog-select-media-player.ts [hunk 1]
```diff
@@ -49,30 +52,31 @@ export class HuiDialogSelectMediaPlayer extends LitElement {
...
+            )}</mwc-list-item
           >
           ${this._params.mediaSources
             .sort((a, b) => compare(computeStateName(a), computeStateName(b)))
             .map(
               (source) => html`
-                <paper-item .itemName=${source.entity_id}
-                  >${computeStateName(source)}</paper-item
+                <mwc-list-item
+                  ?noninteractive=${UNAVAILABLE_STATES.includes(source.state)}
```
Comment (bramkragten) [inline src/panels/media-browser/hui-dialog-select-media-player.ts:66]:
```suggestion
                  .disabled=${UNAVAILABLE_STATES.includes(source.state)}
```

File: src/panels/media-browser/hui-dialog-select-media-player.ts [hunk 2]
```diff
@@ -84,8 +88,8 @@ export class HuiDialogSelectMediaPlayer extends LitElement {
         ha-dialog {
           --dialog-content-padding: 0 24px 20px;
         }
-        paper-item {
-          cursor: pointer;
+        mwc-list-item[noninteractive] {
+          color: var(--disabled-text-color);
         }
```
Comment (bramkragten) [inline src/panels/media-browser/hui-dialog-select-media-player.ts:93]:
```suggestion
```

Comment (zsarnett) [reply src/panels/media-browser/hui-dialog-select-media-player.ts:93]: We should still show the disabled color?

So I want to either class map or ?disabled and use the attribute

Comment (bramkragten) [reply src/panels/media-browser/hui-dialog-select-media-player.ts]: a disabled mwc-list should have disabled color, if not we should set the right vars.

**Pull request merged**